data/Republic Navy Intro.txt

```diff
@@ -0,0 +1,265 @@
+	source
+		government "Republic"
+	destination "Follower"
+	to offer
+		has "Navy Scouting 2: done"
+		not "chosen sides"	
+		random < 40
+	
+	on offer
+		conversation
+			`As you step off your ship, you are greeted by an out of breath Jane Mulgrew. "Captain <last>!" she shouts, "Glad I caught you. I have another job, and this one I couldn't just send you a message for.`
+			choice
+				`	"What is going on?"`
+				`	"Sorry, I'm really busy."`
+					decline
+			`	"We have reason to believe there is a ship loaded with alien tech somewhere in the south," she says. "As I'm sure you've heard, the worlds in this area have been organizing a rebellion. So far they've been peaceful, but if they are arming themselves with alien technology, that could be a major problem for us. Would you be able to track down the ship and scan it?"`
+			choice
+				`	"Sure, that sounds simple enough."`
+				`	"Sorry, that sounds way too dangerous."`
+					decline
+			`	"Fantastic. It's a Falcon named the <npc>, last seen near Yed Prior. If it is alien technology, we'll be the first in the Republic to see it, how exciting.`
+				accept
+	
+	on accept
+		log "Volunteered to scan a ship in Free Worlds space that might be armed with Alien technology."
+	
+	npc "scan outfits"
+		government "Free Worlds"
+		personality staying uninterested
+		fleet
+			names "free worlds capital"
+			variant
+				"Falcon (Heavy)"
+		conversation
+			`As soon as you scan the <npc> its captain hails you. "Excuse me, captain," he says, "mind telling me why you felt the need to probe my ship?"`
+			choice
+				`	"Your ship looked so powerful, I just had to know what kind of gear you had. Some of it looked almost alien."`
+					goto alien
+				`	"I hope to have a ship like that someday, and I just wanted a shopping list of what I would need."`
+					goto equipment
+			label alien
+			`	"Alien, well maybe to someone flying a tub like yours," he says, chuckling. "No, I promise you this is all standard human fare."`
+				goto close
+			label equipment
+			`	He looks awfully pleased with himself. "Yes, she is quite the beauty. I would have never been able to afford anything like this without the Defense Pact's scouting program," he says. "She makes keeping pirates out of our territory as easy as a dream. If you're interested in trading that tub you've got in for something like this, the program should be hiring again soon."`
+				goto close
+			label close
+			`	You cut the comlink and breathe a sigh of relief that the conversation is over.`
+	
+	on complete
+		"assisted Republic" ++
+		dialog `Soon after you land, Jane comes to your ship and pays you <payment> in exchange for your sensor logs regarding <npc>. "No alien tech, I see." She says, sounding dissapointed. "While that would have been a great find, I will admit I'm happy I don't have to figure out how to defend against it. I've let my supervisors know how helpful you've been, hopefully we can work together again soon." `
```

Comment:
`"I see." She says,` -> `"I see," she says,`

The comma splice in the last sentence should be resolved.

Trailing space.